Source/lighting.cpp

```diff
@@ -20,7 +20,7 @@ int VisionId;
 Light Lights[MAXLIGHTS];
 uint8_t ActiveLights[MAXLIGHTS];
 int ActiveLightCount;
-char LightsMax;
+const char LightsMax = 15;

```

Comment:
```suggestion
constexpr char LightsMax = 15;
```

You can probably also move it to the header.